README.md

```diff
@@ -33,29 +33,36 @@ modified for specific [OpenLab](https://github.com/theopenlab) CI system deploym
 
      Using two nodes to  create a new CI system. One is for Master, the other one is for Slave
 
-   - New Slave Mode (`new-slave`)
+   - New Slave for HA Mode(`new-slave`)
 
-     Recovering a new Slave node based on the existing Master node. Before runing with this mode,
-     please check the `inventory/openlab-new-slave.yaml`, you may need to config the new slave
-     `nodepool` and `zuul` hosts IP, also you can choose whether to sync the old **zuul**
-     database to new slave or not.
+     There is not a specific inventory yaml file of this mode, you can use `--new-slave` argument
+     and new slave hsots IPs specified by `--new-ip` argument of the `deploy.py` tool. For an example,
+      you can run following command to deploy a new *slave* part of a OpenLab environment:
+
+     ```bash
+     $ ./deploy.py openlab-ha --new-slave --new-ip zuul02=192.168.5.5 --new-ip nodepool02=192.168.6.6
```

Comment:
So we can new several slaves one time? --new-ip is required argument? Could we only have to type following command simply:
./deploy.py openlab-ha --new-slave zuul02=192.168.5.5 nodepool02=192.168.6.6

Reply:
the `--new-ip` is not only used for `--new-slave`, it can also be used for deployment with new ip address specified. but when specify `--new-slave`, the `--new-ip` is required. please see :82

Reply:
Thanks for clarification.